src/cmds/versions/create.js

```diff
@@ -67,19 +66,45 @@ module.exports = class CreateVersionCommand {
       );
     }
 
-    if (!fork) {
+    let versionList;
+    if (!opts.fork) {
       versionList = await fetch(`${config.get('host')}/api/v1/version`, {
         method: 'get',
         headers: cleanHeaders(key),
       }).then(res => handleRes(res));
     }
 
-    const versionPrompt = promptOpts.createVersionPrompt(versionList || [{}], {
-      newVersion: version,
-      ...opts,
-    });
+    const questions = [
+      {
+        type: 'select',
+        name: 'fork',
+        message: 'Which version would you like to fork from?',
+        skip: () => opts.fork,
+        choices: (versionList || [{}]).map(v => ({ message: v.version, value: v.version })),
+      },
+      {
+        type: 'confirm',
+        name: 'main',
+        message: 'Would you like to make this version the main version for this project?',
+        skip: () => 'main' in opts,
+      },
+      {
+        type: 'confirm',
+        name: 'beta',
+        message: 'Should this version be in beta?',
+        skip: () => 'beta' in opts,
+      },
+      {
+        type: 'confirm',
+        name: 'public',
+        message: 'Would you like to make this version public?',
+        skip() {
+          return 'public' in opts || opts.main || this.enquirer?.answers?.main;
+        },
+      },
+    ];
 
-    const promptResponse = await prompt(versionPrompt);
+    const answers = await enquirer(opts, questions);
```

Comment:
This is probably my only gripe with this approach: could we tweak the patterns defined in `src/lib/enquirer.js` so this matches the usage in the `enquirer` docs? The variable name `enquirer` here is a bit confusing because the `enquirer` object as it's used in the package itself can't be called with the `questions` array like this. Maybe since the default export in `enquirer.js` is  `enquirer.prompt`, we simply rename this default export to prompt?
```suggestion
    const answers = await prompt(opts, questions);
```

Reply:
I'm good with that.